monitoring/defaults_linux.go

```diff
@@ -113,12 +113,10 @@ func DefaultBootConfigParams() health.Checker {
 		BootConfigParam{Name: "CONFIG_VETH"},
 		BootConfigParam{Name: "CONFIG_BRIDGE"},
 		BootConfigParam{Name: "CONFIG_BRIDGE_NETFILTER"},
-		BootConfigParam{
-			// https://cateee.net/lkddb/web-lkddb/NF_NAT_IPV4.html
-			// CONFIG_NF_NAT_IPV4 has been removed as of kernel 5.1
-			Name:             "CONFIG_NF_NAT_IPV4",
-			KernelConstraint: KernelVersionLessThan(KernelVersion{Release: 5, Major: 1}),
-		},
+		// Note: CONFIG_NF_NAT_IPV4 is being removed from newer kernel versions.
```

Comment:
So I'm not sure if we should leave a comment about the removed config or not. The main purpose would be to protect against being re-added, which could be missed and would be caught by the integration tests anyways. So I suspect it's OK to just totally remove the reference to it.